### DISCUSSION TITLE
Is this project still under development?

### DISCUSSION THREAD

**Discussion by michaelslec** [panoply/Liquify#25] [Q&A]
Just the question in the title :)

**Comment by panoply** [comment]
Hey @michaelslec 

Sort of. I invested the better part of 18 months on this project and I still want to complete it. My hope was that given the scale and size I'd be able to attract some developers to help, but that never become a reality. Currently, the project consists of several sub-modules which are being used in vscode-liquid and it spawned the creation of various other tools (like [Aesthetic](https://aesthetic.js.org/) which I actively work on).

Before Shopify began to monopolise OSS tooling for their platform, Liquify was the only project in circulation trying to solve the problems devs faced with theme development. When Shopify began to actively build tools, it was a tad isolating on my end considering this project had already reached a point where the problems they were trying to solve and deliver upon were already solved in Liquify (though not shipped). On some level, I suppose I had convinced myself that the work here would be something that Shopify engineers would like to adapt and given that vscode-liquid had already been in use by thousands of devs/agencies for years before Liquify was even a thing, I really thought and hoped that I'd be able to get some assistance  to get it over the finish line and when that never happened, I lost interest.

However, having seen what the Shopify team have come up with in the last 18 months in the space, I am left scratching my head because even though Liquify and these projects are still not ready for the big time, the approaches I've introduced are better overall and thus there is still reason for me to continue to work on it. The direction Shopify has gone in this nexus is something I don't see myself adopting and though I have moved most merchants off of Shopify, there is still one that uses it and thus there is still reason for me to work on this.

The reason there are long delays between active work on the core of the project comes down to my day-to-day job and time. I don't work in web dev, I work in fashion so it is hard to get devoted time. The last commit was only 4 months back and I'll be making some action in the spring - Another reason for the long time between work on the project is that I set out to produce something I deeply underestimated when it came to Liquid support as a template language. Initially, Liquify was designed to be Shopify theme environment only, but the architecture allowed for me to produce something that would support all variations of Liquid in any environment (11ty, Jekyll etc) and this definitely made things much more difficult. 

TLDR: Yeah.
